packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -3241,7 +3241,9 @@ class _StandardBottomSheetState extends State<_StandardBottomSheet> {
         _kMaxBottomSheetScrimOpacity - scaffold._floatingActionButtonVisibilityValue,
       ));
     } else {
-      scaffold._floatingActionButtonVisibilityValue = 1.0;
+      if (scaffold.widget.floatingActionButton != null) {
+        scaffold._floatingActionButtonVisibilityValue = 1.0;
+      }
```

Comment:
This might just be hacking around the problem and not addressing the root cause, but I'll defer to @Piinks.